src/consts.rs

```diff
@@ -7,6 +7,9 @@ pub const PLAYER_HEIGHT: f32 = PLAYER_SPRITE_HEIGHT - 50.;
 //
 pub const LEFT_BOUNDARY_MAX_DISTANCE: f32 = 380.;
 
+// In this style of game, considering Y is not significant.
+pub const ENEMY_FOV_X: f32 = 800.;
```

Comment:
Just a thought, should this be an enemy stat, or not really?

Anyway, not something that needs to be done for this PR.

Reply:
I think the exact solution is tripping points (so, level metadata).

As far as I remember from Final Fight and similar, as long as the player was inside the screen, they were always chased by on-screen enemies, so the FOV was effectively one screen. However, I think beat'em ups don't work with FOV actually, rather, with tripping points. FOV doesn't work well though, because if there is a camera stop (wave to beat), the next wave of enemies should be placed at least ENEMY_FOV_X (which is currently one screen), which may be too far.

So this is essentially a stopgap solution to avoid enemies running at the player from the whole level :smile:.

The next step, depending on priorities, will be to add tripping points to the level meta.

Reply:
That makes sense. I'll call this resolved, then and go ahead and merge it.